students/arun_nalla/session03/strformat_lab.py

```diff
@@ -11,15 +11,15 @@
 
 #task2
 # alternative formating
-s2 = '(file_00%s : %.2f, %.2e, %.2e)'%s
-print ('Task2')
+s2 = '(file_00%s: %.2f, %.2e, %.2e)'%s
```

Comment:
It looks like I missed this earlier. If `s[0]` is a number with more than 1 digit (e.g., `11`), your string will be something like `file_0011: ...`, which is not what was asked. You'll need to generate the left padding using a format string syntax, not hard-coding a fixed amount in the string like this.

Also recommended is to use `"...".format(...)`, rather than the old `"..." % (...)` style.